### PR TITLE
Add FlipGPU and GaussianBlurGPU benchmarks

### PR DESCRIPTION
**Pull request by jmicota**
## Description
- [ ] Added benchmarks for operators Flip and GaussianBlur on GPU

### What happened in this PR

- Added dali/benchmark/flip_bench.cc
- Added dali/benchmark/gaussian_blur_bench.cc
- Modified dali/benchmark/CMakeLists.txt

### Documentation
- [ ] Existing documentation applies

#### Requirements: N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

File: dali/benchmark/flip_bench.cc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  }
+}
+
+BENCHMARK_DEFINE_F(OperatorBench, FlipGPU)(benchmark::State& st) {
+  int batch_size = st.range(0);
+  int H = st.range(1);
+  int W = st.range(2);
+  int C = st.range(3);
+  int horizontal = 1;
+  int vertical = 1;
```
Comment (JanuszL) [inline dali/benchmark/flip_bench.cc:36]: If these values are not set the perf difference is still the same between big and small batch or not really?

File: dali/benchmark/gaussian_blur_bench.cc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+  }
+}
+
+BENCHMARK_DEFINE_F(OperatorBench, GaussianBlurGPU)(benchmark::State& st) {
+  int batch_size = st.range(0);
+  int H = st.range(1);
+  int W = st.range(2);
+  int C = st.range(3);
+  float sigma = 1.0;
```
Comment (JanuszL) [inline dali/benchmark/gaussian_blur_bench.cc:35]: Same question as in case of flip op.

**Review by JanuszL — APPROVED** [review]

**Review by awolant — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Pull request merged**